libcurl_vendor/CMakeLists.txt

```
@@ -14,64 +14,33 @@ if (POLICY CMP0135)
 endif()
 
 macro(build_libcurl)
-  set(extra_cxx_flags)
+
   set(extra_c_flags)
   if(NOT WIN32)
-    list(APPEND extra_cxx_flags "-std=c++14 -w")
     list(APPEND extra_c_flags "-w")
   endif()
 
-  set(PKGCONFIG_FOR_OPENSSL)
-  if(APPLE)
-    set(PKGCONFIG_FOR_OPENSSL "/usr/local/opt/openssl/lib/pkgconfig")
-  endif()
-
   if(WIN32 AND NOT ${CMAKE_VERBOSE_MAKEFILE})
     set(should_log ON)  # prevent warnings in Windows CI
   else()
     set(should_log OFF)
   endif()
 
   include(ExternalProject)
-  if(WIN32)
-    ExternalProject_Add(curl-7.81.0
-      URL https://github.com/curl/curl/releases/download/curl-7_81_0/curl-7.81.0.tar.gz
-      URL_MD5 9e5e81fc7657eea8dc66672768082c46
-      LOG_CONFIGURE ${should_log}
-      LOG_BUILD ${should_log}
-      CMAKE_ARGS
-        -DENABLE_MANUAL=OFF
-        -DBUILD_TESTING=OFF
-        -DCMAKE_INSTALL_PREFIX=${CMAKE_CURRENT_BINARY_DIR}/libcurl_install
-        -Wno-dev
-      TIMEOUT 600
-    )
-  else()
-    ExternalProject_Add(curl-7.81.0
-      URL https://github.com/curl/curl/releases/download/curl-7_81_0/curl-7.81.0.tar.gz
-      URL_MD5 9e5e81fc7657eea8dc66672768082c46
-      CONFIGURE_COMMAND
-        <SOURCE_DIR>/configure
-        CFLAGS=${extra_c_flags}
-        CXXFLAGS=${extra_cxx_flags}
-        PKG_CONFIG_PATH=$PKG_CONFIG_PATH:${PKGCONFIG_FOR_OPENSSL}
-        --prefix=${CMAKE_CURRENT_BINARY_DIR}/libcurl_install
-        --with-ssl
-        --without-libgsasl
-        --without-libpsl
-        --without-libidn2
-      BUILD_COMMAND $(MAKE)
-      INSTALL_COMMAND $(MAKE) install
-      TIMEOUT 600
-    )
-  endif()
 
-  # The external project will install to the build folder, but we'll install that on make install.
-  install(
-    DIRECTORY
-      ${CMAKE_CURRENT_BINARY_DIR}/libcurl_install/
-    DESTINATION
-      ${CMAKE_INSTALL_PREFIX}/opt/libcurl_vendor
+  ExternalProject_Add(curl-7.81.0
+    URL https://github.com/curl/curl/releases/download/curl-7_81_0/curl-7.81.0.tar.gz
+    URL_MD5 9e5e81fc7657eea8dc66672768082c46
+    LOG_CONFIGURE ${should_log}
+    LOG_BUILD ${should_log}
+    CMAKE_ARGS
+      -DCMAKE_C_FLAGS=${extra_c_flags}
+      -DENABLE_MANUAL=OFF
+      -DBUILD_TESTING=OFF
+      -DCURL_ENABLE_SSL=ON  # default, but just to make clear
+      -DUSE_LIBIDN2=OFF
+      -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
```

I'm not totally sure why it was installed in `opt/libcurl_vendor` before, but let's not change it at this juncture.  So let's make this:

```suggestion
      -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}/opt/libcurl_vendor
```

---

Actually, scratch that.

The whole reason for the previous dance is that we install it into the `build` directory, and then copy it from there into the `install` directory.  That matches what we do with all of our other packages.  So instead we should go back to the previous code where we did:

```
-DCMAKE_INSTALL_PREFIX=${CMAKE_CURRENT_BINARY_DIR}/libcurl_install
```

And then do the install as a separate step.